.gitignore

```diff
@@ -1 +1,2 @@
 vendor/
+.idea/
```

Comment:
I'm happy to add this but in general I prefer to keep IDE related ignores out of project `.gitignores`. Instead I add them to my global git ignore list (https://help.github.com/articles/ignoring-files/) ;)